sdk/core/core-http/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.1.7 (Unreleased)
+## 1.1.7 (2020-09-02)
 
```

Comment:
@sadasant, changelog for token refreshing should be worth mentioning I guess.

Reply:
We already have an entry for #10728.

Reply:
Just had a chat with @sadasant. Since the issue addressed by #10728 has not been released, we replace this entry with the token refresher summary.

Reply:
Perfect